src/common/types/enums/ecdsa.ts

```diff
@@ -0,0 +1,6 @@
+enum ECDSA {
+    ed25519 = 'ed25519',
+    secp256k1 = 'ecdsa-secp256k1',
```

Comment:
```suggestion
    secp256k1 = 'secp256k1',
```

Reply:
I think this suggested change will cause code to break. ripple-keypairs selects `secp256k1` method as `'ecdsa-secp256k1'` in `select(algorithm)`. So that's why I set `secp256k1 = 'ecdsa-secp256k1',` - https://github.com/ripple/ripple-keypairs/blob/07ce053f05e0b14155575de2b60e08a98c09ed1b/src/index.ts#L101

Reply:
I created a ticket to track this change - https://ripplelabs.atlassian.net/browse/DEVX-227